src/api/entities/ApiRole.ts

```diff
@@ -25,12 +25,14 @@ export class ApiRole extends BaseModel {
   declare id: number;
   declare roleName: string;
   declare externalName: string;
+  declare disabled: boolean;
 }
 
-export type ApiRoleDTO = ModelObjectOpt<ApiRole>;
+export type ApiRoleDTO = Omit<ModelObjectOpt<ApiRole>, 'disabled'>;
 
 export const ApiRoleSchema = z.object({
   id: z.number(),
   roleName: z.string(),
   externalName: z.string(),
+  disabled: z.boolean(),
```

Comment:
I don't think we need this line?

Reply:
Yeah we don't atm. I would think though it's better to have for later? Happy to remove if not really a point.

Reply:
I think remove it if we aren't using it now. We may never use it?

Reply:
Sounds good!